### PR TITLE
Jaini: Version 2.000; ttfautohint (v1.8.4.7-5d5b) added

### PR DESCRIPTION
**Pull request by yanone**
b617ba7: [gftools-packager] Jaini: Version 2.000; ttfautohint (v1.8.4.7-5d5b) added

* Jaini Version 2.000; ttfautohint (v1.8.4.7-5d5b) taken from the upstream repo https://github.com/EkType/Jaini at commit https://github.com/EkType/Jaini/commit/c7bc27878832b1f56a7224fdb07100d39503e89f.

### CODE REVIEW

File: ofl/jaini/DESCRIPTION.en_us.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<p>
+    Jaini and Jaini Purva (both Devanagari) are typefaces based on the calligraphic style of the Jain Kalpasutra
+    manuscripts,
+    particularly referencing a manuscript from 1503 CE. The manuscript style has several unique features not seen in the
```
Comment (vv-monsalve) [inline ofl/jaini/DESCRIPTION.en_us.html:4]: @yanone, Same comment as in Jaini Purna: the formatting of the description file looks broken, please adjust it following this https://github.com/fonttools/fontbakery/issues/2811

**Review by vv-monsalve — CHANGES_REQUESTED** [review]

**Comment by vv-monsalve** [comment]
I've added a [coment on the similarities between Jaini and Jaini Purva in the font issue](https://github.com/google/fonts/issues/3830#issuecomment-1911127968)

**Comment by emmamarichal** [comment]
@yanone, same question, could you edit that PR so we can merge it?

**Comment by yanone** [comment]
Updated

## Jaini: Version 2.000; ttfautohint (v1.8.4.7-5d5b) added

---
 97b1c87: [gftools-packager] Jaini: Version 2.000; ttfautohint (v1.8.4.7-5d5b) added

* Jaini Version 2.000; ttfautohint (v1.8.4.7-5d5b) taken from the upstream repo https://github.com/EkType/Jaini at commit https://github.com/EkType/Jaini/commit/c7bc27878832b1f56a7224fdb07100d39503e89f.

**Comment by yanone** [comment]
The smart_dropout check isn't supposed to fail. It passes when I run fontbakery with just this check and I investigated the font and found it to be correct. I reported the [issue](https://github.com/fonttools/fontbakery/issues/4525) and ask that this FAIL gets ignored.

**Comment by yanone** [comment]
I emailed Ektype about filling the categorization form just now.

**Comment by yanone** [comment]
Still waiting for the categorization by the authors...

**Comment by emmamarichal** [comment]
status:
these warns should be avoid and updated in sources:
⚠️ **WARN** Check copyright namerecords match license file.
⚠️ **WARN** License URL matches License text on name table?

**Comment by yanone** [comment]
Updated

**Review by vv-monsalve — APPROVED** [review]

**Comment by vv-monsalve** [comment]
@yanone I'm merging this here. However, a minisite_url is expected for standard commissioned projects. Please confirm if there is going to be one for the Jaini fonts, and if so, it needs to be included in the `METADATA.pb` file.

**Pull request merged**